Gemfile

```diff
@@ -6,5 +6,7 @@ source 'https://rubygems.org'
 gemspec
 
 group :development do
+	gem 'rake', '~> 12.0'
+	gem 'rspec'
```

Comment:
nit: formatting to match line below (are these tabs?)

Reply:
yeap fixed.,\